x.py

```diff
@@ -216,16 +216,18 @@ def write_version(release_version: str) -> str:
     return version
 
 
-def package_source(release_version: str) -> None:
+def package_source(release_version: str, release_candidate_number: int) -> None:
```

Comment:
Should it be an optional ( https://docs.python.org/3/library/typing.html#typing.Optional ) ?